packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -99,7 +101,19 @@ const OpenQueueCard = ({ queue, isTA, updateQueueNotes }: OpenQueueCard) => {
     <PaddedCard
       headStyle={{ background: "#F3F5F7" }}
       title={staffList.map((staffMember) => staffMember.name).join(", ")}
-      extra={<ExtraText>//TODO: 3:00 - 5:00</ExtraText>}
+      extra={
```

Comment:
This can be done later, but since we're converting to local time, it might cause confusion for students who assume all the times are EST. Maybe we could add a hover popup that explains these times are displayed in your local time?

Reply:
added a TODO